### PR TITLE
Fix style issues in `Eateries/Controllers`

### PR DESCRIPTION
**Pull request by whoiswillma**
yeah

### CODE REVIEW

File: Eatery/Controllers/Legacy/Eateries/CampusEateryMenuViewController.swift [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//swiftlint:disable all
```
Comment (ywsang) [inline Eatery/Controllers/Legacy/Eateries/CampusEateryMenuViewController.swift:1]: assuming disabling b/c these are legacy files?

File: Eatery/Controllers/Look Ahead/LookAheadViewController.swift [hunk 2]
```diff
@@ -181,8 +181,14 @@ class LookAheadViewController: UIViewController {
             make.edges.equalToSuperview()
         }
 
-        tableView.register(LookAheadTableViewCell.self, forCellReuseIdentifier: CellIdentifier.eatery.rawValue)
-        tableView.register(LookAheadHeaderView.self, forHeaderFooterViewReuseIdentifier: HeaderIdentifier.header.rawValue)
+        tableView.register(
+            LookAheadTableViewCell.self,
+            forCellReuseIdentifier: CellIdentifier.eatery.rawValue
+        )
```
Comment (ywsang) [inline Eatery/Controllers/Look Ahead/LookAheadViewController.swift:187]: just curious -- is this a line length thing or splitting up arguments across lines?

**Review by ywsang — APPROVED** [review]
looks good!

File: Eatery/Controllers/Legacy/Eateries/CampusEateryMenuViewController.swift (hunk 1 above)
Comment (whoiswillma) [reply Eatery/Controllers/Legacy/Eateries/CampusEateryMenuViewController.swift:1]: Yup

File: Eatery/Controllers/Look Ahead/LookAheadViewController.swift (hunk 2 above)
Comment (whoiswillma) [reply Eatery/Controllers/Look Ahead/LookAheadViewController.swift:187]: Line length: I tried not to split arguments unless it was necessary

**Pull request merged**